Documents/LearnJs/Constructor(prototype)/js/script.js

```diff
@@ -0,0 +1,95 @@
+//1
+function Car(make, model, year, color, price) {
+    this.make = make;
+    this.model = model;
+    this.year = year;
+    this.color = color;
+    this.price = price;
+}
+
+function Garage() {
+    this.cars = [];
+}
+
+Garage.prototype.addCar = function (make, model, year, color, price) {
+    const newCar = new Car(make, model, year, color, price);
+    this.cars.push(newCar);
+    return "New car was added: " + make + " " + model;
+}
+
+const myGarage = new Garage();
+
+
+function addCarToGarage() {
```

Comment:
this function should be a part of function constructor Garage. As example addCar.